### ISSUE TITLE
Copy saleor app token to clipboard

### ISSUE THREAD

**Issue by witoszekdev** [saleor/saleor-cli#550]
Our pretty-printing looks nice, but unfortunately double-click + CTRL + C on some teminals copies unwanted spaces, which could lead to frustration if left unnoticed by developer.

We could add a flag to support the value to clipboard or allow invocation of the command that makes the output pipeable:

```bash
saleor app token <environment> <app_id> | pbcopy
```

**Comment by zaiste** [comment]
@witoszekdev it's done and soon will be available in `1.23.0` 
```sh
saleor app token --environment XXX --app-id YYY --short | pbcopy
```

**Comment by zaiste** [comment]
This is released in `1.23.0-rc.1`
